examples/features/webhook-smartvf/README.md

```diff
@@ -0,0 +1,150 @@
+# Client requests for postgresql service over SmartVF
+
+This example demonstrates how Postgres-client can get connectivity to Postgres-server deployment via NSM over SmartVF interfaces.
+Client pod and server deployment located on different nodes. The nodes must support SmartVF interface pool as resources.
+
+
+## Requires
+
+Make sure that you have completed steps from [ovs](../../ovs) setup
+
+## Run
+
+Note: Admission webhook is required and should be started at this moment.
+```bash
+WH=$(kubectl get pods -l app=admission-webhook-k8s -n nsm-system --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
+kubectl wait --for=condition=ready --timeout=1m pod ${WH} -n nsm-system
+```
+
+1. Create test namespace:
+```bash
+NAMESPACE=($(kubectl create -f https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/f51e483c361569efcd41934b403c33220dcf34e4/examples/features/namespace.yaml)[0])
```

Comment:
I think you need to change `f51e483c361569efcd41934b403c33220dcf34e4` to
`9948922016185ac53315d0f847b231ae8db14ac7` to pass ci, because your commits are based on [this commit](https://github.com/Nordix/nsm-deployments-k8s/commit/74dce5a3481d1a0c6909758c699ea0a5f4b68f77) and there is used `9948922...` for `to-local`- `to-ref` scripts

Reply:
Still the same issue. I have changed to point to my latest dependent parent commit but still the CI is failing. Any other ideas ?

Reply:
Got it. it seems we missed one thing...
ci is used `checkout` github action. And when is starts, it creates merge-commit if `upstream` `main` branch is already far away.
So, as workaround for now I can suggest:
1. Rebase your PR to upstream main.
2. Change your refs to refs from main. Current is `ec20b7f28d26571b68497d4030fe55408286218d` ([commit](https://github.com/networkservicemesh/deployments-k8s/commit/ef089a748a45fb1fd91dcf8fa40becae349d3239))

Reply:
@glazychev-art ok it seems that the issue is solved. Thanks a lot!